api/ca.proto

```diff
@@ -55,3 +60,9 @@ message GetRootCACertificateRequest {}
 message GetRootCACertificateResponse {
 	bytes certificate = 1;
 }
+
+message GetUnlockKeyRequest {}
+
+message GetUnlockKeyResponse {
+	bytes unlock_key = 1;
```

Comment:
@lvh: Thank you for the good explanation. This sounds similar to the justification I've seen for the security of hash-based signatures against quantum attacks.